### PR TITLE
contact us page update

### PR DESCRIPTION
**Pull request by IsuruLakshan97**
buttton added with some minor changes

### CODE REVIEW

File: debug.log [hunk 1]
```diff
@@ -0,0 +1 @@
+[1106/192439.086:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```
Comment (luk3Sky) [inline debug.log]: Don't track these files
@IsuruLakshan97 add *.log in the .gitignore file

**Review by luk3Sky — CHANGES_REQUESTED** [review]
@IsuruLakshan97 
Format your code before pushing to the remote.
Run the command `npm run format` in the project directory if you have Node.is installed, otherwise install it and then run.

File: debug.log (hunk 1 above)
Comment (IsuruLakshan97) [reply debug.log]: done

**Review by luk3Sky — APPROVED** [review]
Accepted

**Pull request merged**